docs/whats-new/preview-4.md

```diff
@@ -692,8 +692,33 @@ This release we've enabled a Visual Studio publish experience for .NET Aspire ap
 
 With so many similarities between Container Apps' support for [add-on services](/azure/container-apps/services) like PostgreSQL, Kafka, and Redis and the support offered by mirroring .NET Aspire components ([PostgreSQL](/dotnet/aspire/database/postgresql-component?tabs=dotnet-cli), [Kafka](/dotnet/aspire/messaging/kafka-component?tabs=dotnet-cli), and [Redis](/dotnet/aspire/caching/stackexchange-redis-component?tabs=dotnet-cli), for example), targeting Azure Container Apps as our **first** publishing destination was an easy decision. You'll be able to use Azure Container Apps as your development publishing target out-of-the-box using Visual Studio Preview's upcoming 17.10 preview 3 release.
 
-### Considerations during Visual Studio 17.10 preview 3
+### Considerations during Visual Studio 17.10 Preview 2
 
 In this release, we have two important points you'll want to consider. The Visual Studio publish experience currently lacks support for parameter prompting. We'll add this in a subsequent release.
 
 The second, and more important thing to note, is that you'll need to perform an `azd auth login` before the Visual Studio publish features will work. Once you do the `azd auth login`, you'll be able to right-click publish to any Azure subscription you can access. We're working on this integration and plan to release an integrated auth flow experience from Visual Studio to `azd` prior to Visual Studio 17.10 preview 4. Once you do the `azd auth login`, you'll be good to go.
+
+## Known Issues
+
+Below are some known issues with .NET Aspire Preview 4.
+
+### Console logs for a container do not show up when using Podman
+
+When using Podman, the console logs for a container do not show up in the Dashboard. See issues [aspire#2701](https://github.com/dotnet/aspire/issues/2701) and [aspire#2705](https://github.com/dotnet/aspire/issues/2705) for more details. This issue happens because some assumptions were made about the availability of Docker as mentioned [here](https://github.com/dotnet/aspire/issues/2701#issue-2173228010).
```

Comment:
```suggestion
When using Podman, the console logs for a container don't show up in the .NET Aspire dashboard. For more information, see issues [aspire#2701](https://github.com/dotnet/aspire/issues/2701) and [aspire#2705](https://github.com/dotnet/aspire/issues/2705). This issue is because some assumptions were made about the availability of Docker, see GitHub Aspire repository [issue #2701 comment](https://github.com/dotnet/aspire/issues/2701#issue-2173228010).
```